src/strapi/index.ts

```diff
@@ -32,21 +37,24 @@ export const normalize = (content) => {
 }
 
 const normalizeObject = (content) => {
-  let normalizedAttributes = {}
-  let normalizedRelations = {}
+  let normalizedData = {}
 
   if (content.attributes) {
-    normalizedAttributes = normalize(content.attributes)
-  }
-
-  if (content.relations) {
-    normalizedRelations = normalize(content.relations)
+    normalizedData = normalize(content.attributes)
   }
 
-  return {
+  const result = {
     id: content.id,
-    ...normalizedAttributes,
-    ...normalizedRelations,
+    ...normalizedData,
     ...content,
   }
+
+  // Remove undefined values from the result object
+  Object.keys(result).forEach(key => {
+    if (result[key] === undefined) {
+      delete result[key];
+    }
+  });
```

Comment:
We use a technique like this in code-generators to remove `null` and `unndefined`